client/src/Components/Cam.js

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
```

Comment:
Why do we need to import the default `React` here?  Is there plan to reference it later on?  Why not destructure all of the necessary methods off of it, like `Component`?